<ui version="4.0" >
 <class>imodvViewsForm</class>
 <widget class="QWidget" name="imodvViewsForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>303</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>3dmodv Views</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QListWidget" name="viewListBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0" >
        <bool>false</bool>
       </property>
       <property name="selectionRectVisible" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="storeButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>80</width>
           <height>32767</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip" >
          <string>Save current model properties in selected view</string>
         </property>
         <property name="text" >
          <string>Store</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="revertButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>80</width>
           <height>32767</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip" >
          <string>Return to the stored values of the currently selected view</string>
         </property>
         <property name="text" >
          <string>Revert</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>80</width>
           <height>32767</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip" >
          <string>Remove the currently selected view from the list</string>
         </property>
         <property name="text" >
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>80</width>
           <height>32767</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip" >
          <string>Add a new view to the list with the properties of current display</string>
         </property>
         <property name="text" >
          <string>New View</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>80</width>
           <height>32767</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip" >
          <string>Save model to file, with all view modifications</string>
         </property>
         <property name="text" >
          <string>Save Model</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoStoreBox" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip" >
          <string>Automatically store changes in the current view when switching to another view, saving model, etc.</string>
         </property>
         <property name="text" >
          <string>Autostore</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="viewLabelEdit" >
     <property name="focusPolicy" >
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="toolTip" >
      <string>Change the name of the currently selected view</string>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="Line1_2" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="doneButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>70</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Close dialog box</string>
       </property>
       <property name="text" >
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>70</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Open help window</string>
       </property>
       <property name="text" >
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>autoStoreBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>imodvViewsForm</receiver>
   <slot>autostoreToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewLabelEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>imodvViewsForm</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewLabelEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>imodvViewsForm</receiver>
   <slot>newLabelEntered()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doneButton</sender>
   <signal>clicked()</signal>
   <receiver>imodvViewsForm</receiver>
   <slot>donePressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpButton</sender>
   <signal>clicked()</signal>
   <receiver>imodvViewsForm</receiver>
   <slot>helpPressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>storeButton</sender>
   <signal>clicked()</signal>
   <receiver>imodvViewsForm</receiver>
   <slot>storePressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>revertButton</sender>
   <signal>clicked()</signal>
   <receiver>imodvViewsForm</receiver>
   <slot>revertPressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>imodvViewsForm</receiver>
   <slot>deletePressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newButton</sender>
   <signal>clicked()</signal>
   <receiver>imodvViewsForm</receiver>
   <slot>newPressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>imodvViewsForm</receiver>
   <slot>savePressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewListBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>viewLabelEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>295</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel" >
     <x>116</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
